boards/waveshare/rp2040_plus/doc/index.rst

```diff
@@ -0,0 +1,98 @@
+.. zephyr:board:: rp2040_plus
+
+Overview
+********
+
+RP2040-Plus, A Low-Cost, High-Performance Pico-Like MCU Board Based On Raspberry Pi Microcontroller RP2040
+including a battery charger.
+
+Hardware
+********
+- Dual core Arm Cortex-M0+ processor running up to 133MHz
+- 264KB on-chip SRAM
+- 2MB on-board QSPI flash with XIP capabilities
+- 26 GPIO pins
+- 3 Analog inputs
+- 2 UART peripherals
+- 2 SPI controllers
+- 2 I2C controllers
+- 16 PWM channels
+- USB 1.1 controller (host/device)
+- 8 Programmable I/O (PIO) for custom peripherals
+- On-board LED
+- 1 Watchdog timer peripheral
+- on-board battery charger
+
+
+.. figure:: img/rp2040-plus.webp
```

Comment:
rename filename to `rp2040_plus.webp`
and remove `figure::` section.

`img/[board_name].webp` is automatically handled by the document generator.

https://builds.zephyrproject.io/zephyr/pr/89527/docs/boards/waveshare/rp2040_plus/doc/index.html